### ISSUE TITLE
Fullscreen: Missing Title Bar Buttons

### ISSUE THREAD

**Issue by bjpcorp** [mafintosh/playback#31]
Title Bar Buttons missing in fullscreen mode.

Great app!

**Comment by mafintosh** [comment]
I think this is by design. /cc @freeall ?

**Comment by freeall** [comment]
Yes, this is by design. I don't know any app which have the title bar when in fullscreen. Or am I mistaken?

**Comment by bjpcorp** [comment]
I think most apps have a title bar, it slides out of view in fullscreen/maximise mode. Playback displays a grey coloured title bar missing the title bar buttons in fullscreen mode.

![screen-shot-2015-05-05-at-16 55](https://cloud.githubusercontent.com/assets/1779524/7477115/b189007e-f349-11e4-8a8b-31034293d455.jpg)

**Comment by freeall** [comment]
Hmm, it doesn't do that on my screen. Let me check a few things. It's supposed to not have that top grey bar.

**Comment by bjpcorp** [comment]
Screenshot on 27" iMac Yosemite 10.10.3 / Playback 0.18.0

**Comment by mafintosh** [comment]
The grey bar sounds like an electron issue. @freeall that reminds me that we should do a release that bump electron to the latest version
